xcube_cmems/cmems.py

```diff
@@ -140,7 +112,32 @@ def get_all_dataset_ids(self) -> Dict[str, Any]:
                             split_paths = path.split('/')
                             self.opendap_dataset_ids[split_paths[-1]] = \
                                 record.title
+
+    async def get_csw_records_concurrently(self, csw) -> None:
+        """
+        get csw records concurrently
+        """
+        tasks = []
+        for rec in range(0, 265, 50):
```

Comment:
Where does the number 265 come from? Can you introduce telling variable names for this and 50?

Reply:
I agree. Please avoid magic numbers in code.

Reply:
Introduced variables